tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot

```diff
@@ -16,11 +18,12 @@ Test Teardown       End Web Billing Metrics Test
 ${METRIC_RHODS_CPU}                 cluster:usage:consumption:rhods:cpu:seconds:rate1h
 ${METRIC_RHODS_CPU_BEFORE_1.5.0}    cluster:usage:consumption:rhods:cpu:seconds:rate5m
 ${METRIC_RHODS_UNDEFINED}           cluster:usage:consumption:rhods:undefined:seconds:rate5m
-
+${METRIC_RHODS_ACTIVE_USERS}        cluster:usage:consumption:rhods:active_users
+${telmeter_url}                     https://telemeter-lts-dashboards.datahub.redhat.com/api/datasources/proxy/2/api/v1/
```

Comment:
typo in the name, it should be `telemeter`.
If the URL never changes, wouldn't it be better to have it in `test-variables.yml`?

Reply:
The url will be slightly different for getting data from clusters in stage and production:

stage: https://telemeter-lts-dashboards.datahub.redhat.com/api/datasources/proxy/2/api/v1/
production: https://telemeter-lts-dashboards.datahub.redhat.com/api/datasources/proxy/1/api/v1/

@pablofelix is there any keyword to know if we are in OSD stage or production?